lab_8_llm/main.py

```diff
@@ -34,6 +25,7 @@ class RawDataImporter(AbstractRawDataImporter):
     """
     A class that imports the HuggingFace dataset.
     """
+    _raw_data: DataFrame
```

Comment:
Are you sure this announcement is necessary?

Reply:
They were in the last project, I will try without them

Reply:
update: like previous project lab_7, without it mypy tests crash

Reply:
Need to add a check: `_raw_data` containt `None`? If yes, then rise error

Reply:
okay